project_resource_calendar/__manifest__.py

```diff
@@ -26,6 +26,9 @@
         'views/resource_view.xml',
         'views/room_view.xml',
         'views/instrument_view.xml',
+        'templates/assets.xml',
+        'report/calendar_event_report_view.xml',
```

Comment:
Better to rename the file to calendar_event_weekly_report we will have also another report with this name in another PR